_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/styles/rhd.scss

```diff
@@ -419,6 +419,14 @@ img {
   p { margin-bottom: 0; }
 }
 
+code,
+kbd,
+pre,
+samp {
+  font-family: monospace, serif;
+  font-size: 1em;
```

Comment:
Shouldn't this be a rem instead of em?

Reply:
In this case `em` is appropriate since the size should be relative to the sizing of whatever container it might be in.